scripts/run-gradually-deprecated.sh

```diff
@@ -56,6 +56,35 @@ function runGraduallyDeprecatedFunctions {
       exit 1
   }
 
+  # check for new combined CreateUpdate methods
+  line=$(grep -H -n "Create:.*CreateUpdate," "$f" -m1)
+  if [ "$line" != "" ];
+  then
+    git diff --diff-filter=AMRC origin/main -U0 "$f" | grep -q "+.*Create:.*CreateUpdate," && {
+      echo "$line"
+      echo "New Resources should no longer use combined CreateUpdate methods, please"
+      echo "split these into two separate Create and Update methods."
+      echo ""
+      echo "Existing resources can continue to use combined CreateUpdate methods"
+      echo "for the moment - but over time these will be split into separate Create and"
+      echo "Update methods."
+      exit 1
+    }
+  fi
+
+  # check for d.Get inside Delete
+  deleteFuncName=$(grep -o "Delete: .*," "$f" -m1 | grep -o " .*Delete")
```

Comment:
Worth noting this will only work for non-Typed resources. The newer typed resources should possibly be also checked for accessing `.Get()`. Probably worth tackling in a subsequent PR?